addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -3,17 +3,17 @@
 
     <t t-name="mail.ChatWindow" owl="1">
         <t t-if="chatWindow">
-            <div class="o_ChatWindow bg-white" t-attf-class="{{ className }}" tabindex="0" t-att-data-visible-index="chatWindow.visibleIndex"
-                t-att-class="{
-                    'o-focused': chatWindow.isFocused,
-                    'o-folded': chatWindow.isFolded,
-                    'o-fullscreen': props.isFullscreen,
-                    'o-mobile': messaging.device.isMobile,
-                    'o-new-message': !chatWindow.thread,
-                }" t-att-style="chatWindow.componentStyle" t-on-keydown="_onKeydown" t-on-focusout="_onFocusout" t-att-data-chat-window-local-id="chatWindow.localId" t-att-data-thread-local-id="chatWindow.thread ? chatWindow.thread.localId : ''" t-ref="root"
+            <div class="o_ChatWindow position-absolute bottom-0 d-flex flex-column rounded-lg-top bg-white" t-attf-class="{{ className }}" tabindex="0" t-att-data-visible-index="chatWindow.visibleIndex" t-att-class="{
+                'o-focused': chatWindow and chatWindow.isFocused,
+                'o-folded': chatWindow and chatWindow.isFolded,
+                'o-fullscreen w-100 h-100': props.isFullscreen,
+                'o-mobile position-fixed': messaging and messaging.device.isMobile,
+                'mw-100 mh-100': !messaging and !messaging.device.isMobile,
+                'o-new-message': chatWindow and !chatWindow.thread,
```

Comment:
Still some unnecessary guards.
- No need to `chatWindow and` (it's covered by top-level `t-if="chatWindow"`)
- No need to `messaging and` (it's also implicitly covered by top-level `t-if="chatWindow"`)